packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/Relations.tsx

```diff
@@ -116,11 +117,25 @@ interface Relation extends Pick<RelationResult, 'documentId' | 'id' | 'locale' |
   label: string;
   position?: RelationPosition;
   __temp_key__: string;
+  apiData?: {
+    documentId: RelationResult['documentId'];
+    id: RelationResult['id'];
+    locale?: RelationResult['locale'];
+    position: RelationPosition;
+  };
 }
 
 interface RelationsFieldProps
   extends Omit<Extract<EditFieldLayout, { type: 'relation' }>, 'size' | 'hint'>,
-    Pick<InputProps, 'hint'> {}
+    Pick<InputProps, 'hint'> {
+  documentModel?: string;
+  document?: ReturnType<UseDocument>['document'];
+  setCurrentDocument?: (newDocument: {
+    documentId: string;
+    model: string;
+    collectionType: string;
+  }) => void;
```

Comment:
I don't think we use these props anymore, or we should since we can get the values from context

Reply:
Actually on my branch I already removed the unnecessary props

Reply:
ok I am gonna remove them